### ISSUE TITLE
Slide level fine-tuning loss function and activation function for Binary Classification

### ISSUE THREAD

**Issue by shuangyanl** [prov-gigapath/prov-gigapath#61]
Hello, I noticed that in the slide level fine-tuning implementation, the `CrossEntropyLoss` loss function and the `Softmax` activation function are used in binary classification. This seems to be different from the common practice of binary classification, where `BCEWithLogitsLoss` and `sigmoid` functions are used instead.

Are there particular reasons for these choices? 

Relevant code:
```
finetune/utils.py L310: 
...
elif task_setting == 'multi_class' or task_setting == 'binary':
        loss_fn = torch.nn.CrossEntropyLoss() 
...
```
```
finetune/training.py L213:
...
elif task_setting == 'multi_class' or task_setting == 'binary':
    Y_prob = torch.softmax(logits, dim=1).cpu()
...
```

**Comment by HanwenXuTHU** [comment]
Hi, this is a good question! Actually when using the binary_cross_entropy loss, people usually predefine the positive class. We used the cross_entropy here because sometimes it's hard to say which class should be treated as positive, such as LUAD v.s. LUSC. But you're right we can just specify one class and run with bce loss. I don't expect this will have too much influence on the performance. Let me know if that's not the case.

**Comment by shuangyanl** [comment]
Hi, thank you for your reply, interesting point. I would assume for the mentioned LUAD and LUSC problem, it was treated as a multi class classification so cross_entropy was used. However, in the context of single mutation binary classification problem, normally a label value '1' is given to mutant class while '0' is given to 'wild type' class. 

In this case, how do I specify the `label_dict` in the task config yml file and 'label' column in the dataset csv files as required to run the slide level fine-tuning tasks?
